### ISSUE TITLE
not show "save to playlist"

### ISSUE THREAD

**Issue by gjlmotea** [TheRealJoelmatic/RemoveAdblockThing#6]
Nice work!
But, there are side effects that can cause the "save to playlist" interface not to pop up.

**Comment by JoseErnestoOnGithub** [comment]
I was experiencing the same issue. Here's the video of the issue (GIF only, ignore the watermark):
![Honeycam 2023-10-12 18-56-07](https://github.com/TheRealJoelmatic/RemoveAdblockThing/assets/115112109/092245e4-1113-4d99-b102-87783bc5e29d)

**Comment by Zacchino** [comment]
"Save" (video to playlist) is still blocked by this script: Anyone has a fix?

**Comment by TheRealJoelmatic** [comment]
Try out the latest version. Hopefully, it should be fixed for most popups.

**Comment by Zacchino** [comment]
> Try out the latest version. Hopefully, it should be fixed for most popups.

It works perfectly now! I see how you refined the code. Thank you so, so much for this amazing work and for your generosity!

I fixed some aesthetic stuff in my version of this last release:
- Title written twice on Line 52 or maybe 71 IIRC
- I also renamed the script to “YouTube Adblock Warning Remover” — easier to find when you have several YouTube related scripts, and easier to get what it does in a few words / keywords that people might use to look for it IMO… Hopefully this makes sense too ;)
- And I added a link to this Github Repo in the // @namespace info… not sure if that’s where it goes, but it’s really useful to keep track of the script’s source.
